data/CA_1/costs.csv

```diff
@@ -4,7 +4,7 @@ bat_e,all,2016,fix,|,3020,0
 bat_e,all,2016,var,|,0,0
 bat_in,all,2016,cap,|,280000,0
 bat_in,all,2016,fix,|,2800,0
-bat_in,all,2016,var,|,0,0
+bat_in,all,2016,var,|,0.01,0
```

Comment:
small variable cost to prevent charge and discharge at the same time in times of renewable overproduction (Excess energy was able to be charged and discharged in the same hour without additional costs)

Reply:
Nice, I always was wondering if degeneracy was occuring.

Reply:
Just as a note: Did you add a sentence about this to the documentation?

Reply:
Good thought. Added it ;)